{{cookiecutter.project_slug}}/clients/vue3/src/styles/abstracts/headings.css

```diff
@@ -0,0 +1,41 @@
+/* Headings & Text */
+
+/* h1 headers */
+.heading {
```

Comment:
To do: get John to take a look at these headers to see what naming conventions he'd like to use as a default and we can name them accordingly